### PR TITLE
Add License file

### PR DESCRIPTION
**Pull request by avirmani**
Add License file to Database-Plugins

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,232 @@
+Copyright © 2014-2021 Cask Data, Inc.
```
Comment (snehalakshmisha) [inline LICENSE.txt:1]: Should this still be Cask Data?

Comment (avirmani) [reply LICENSE.txt:1]: All current licenses are that way, hence kept it the same.

Comment (greeshmaswaminathan) [inline LICENSE.txt:1]: Is it intentional to have Cask copyright with Apache license 2 ?

Comment (avirmani) [reply LICENSE.txt:1]: Same license in cdap and other repos as well - https://github.com/cdapio/cdap/blob/develop/LICENSE.txt

**Review by albertshau — APPROVED** [review]

**Pull request merged**